doc/articles/features/windows-ui-storecontext.md

```diff
@@ -0,0 +1,24 @@
+---
+uid: Uno.Features.StoreContext
+---
+
+# Store Context
+
+> [!TIP]
+> This article provides Uno-specific information for the `Windows.Services.Store.StoreContext` namespace. For a comprehensive overview of this feature and detailed usage instructions, refer to the official documentation for [Windows.Services.Store.StoreContext Namespace](https://learn.microsoft.com/uwp/api/Windows.Services.Store.StoreContext).
+
+## In-App Review
+
+The in-app review feature is currently supported only on Android through Google Play.
+
+### Google Play Integration
+
+For Google Play support, make sure to add the `Uno.WinUI.GooglePlay` package to your project. This package is available on [nuget.org](https://www.nuget.org/packages/Uno.WinUI.GooglePlay).
```

Comment:
Was there an associated Uno Feature with the package for Uno.Sdk? In that case it's best to document that for Uno.Sdk users

Reply:
Yep, I was thinking the same this morning, though I don’t think we’ve added it yet. It would be very nice indeed 👍

Reply:
Yes it was not added, but certainly makes sense. Do you want to try to make a PR for that @morning4coffe-dev ?

Reply:
@MartinZikmund Already in progress 😊: https://github.com/unoplatform/uno/pull/18613. Let me know if the changes are sufficient or if more adjustments are needed.